tutorials/vue/scroll.md

```diff
@@ -0,0 +1,83 @@
+To create a custom scroll event in Vue, you must create a Custom Directive.
+Directives must be registered before the Vue instance. If you wish to only
+use a directive locally, components accept a `directives` option.
+
+<div id="app">
+  <h1>Scroll and Watch</h1>
+  <div
+    v-scroll="handleScroll"
+    style="transition: 1.5s all cubic-bezier(0.39, 0.575, 0.565, 1); opacity: 0;"
+  >
+    <p>Surprise!</p>
```

Comment:
This is counter-intuitive because 'Surprise' shows up when you scroll more than 50px down, but then doesn't disappear again when you scroll up. Probably better to just display the current value of `window.scrollY` here instead of using a transition.